resources/sass/global/_variables.scss

```diff
@@ -24,7 +22,40 @@ $orange: #f6993f;
 $yellow: #ffed4a;
 $green: #38c172;
 $teal: #4dc0b5;
-$cyan: #6cb2eb;
-$i-white: #FFFFFF;
-$i-blue: #2699FB;
-$primary: $i-blue;
+
+$primaryColor: rgb(38,153,251);
+$primaryColor--light: lighten($primaryColor, 0.6);
+$primaryColor--med: rgb(16, 109, 188);
+$primaryColor--dark: darken($primaryColor, 0.8);
+$primaryColor--shadow: rgba(38, 153, 251, 0.2);
+
+$secondaryColor: #FFFFFF;
+$secondaryColor--light: lighten($secondaryColor, 0.6);
+$secondaryColor--dark: darken($secondaryColor, 0.8);
+
+// Shades (Standard Shades)
+$white: rgb(245, 245, 245);
+$white--light: rgb(255,255,255);
```

Comment:
One last little small thing, is to make sure there are spaces after each comma.

Reply:
ayy